common/src/main/java/com/wynntils/features/ui/ProfessionHighlightFeature.java

```diff
@@ -182,6 +183,19 @@ public boolean mouseClicked(double mouseX, double mouseY, int button) {
 
             ProfessionType profession = feature.getSelectedProfession();
 
+            // For crafting stations, the only option should be the profession type for the station
+            if (Models.Container.getCurrentContainer() instanceof CraftingStationContainer craftingStation) {
+                if (profession == null) {
+                    profession = craftingStation.getProfessionType();
+                } else {
```

Comment:
```suggestion
                } else if (profession != craftingStation.getProfessionType()){
```
Otherwise this logic would nullify the currently selected profession if it matches the crafting station, no?

Reply:
This will cause you to never be able to disable the highlight which wasn't the original intent but I'll be changing it to work like this now anyway